src/opnsense/mvc/app/models/OPNsense/Base/Menu/Menu.xml

```diff
@@ -5,7 +5,6 @@
             <Root url="/" visibility="hidden"/>
             <RootArgs url="/?*" visibility="hidden"/>
         </Dashboard>
-        <License order="1" url="/license.php" cssClass="fa fa-balance-scale fa-fw"/>
```

Comment:
I'd leave this here modified for the new URL unless we want to create a separate ACL page too.. the general reasoning is "should it deinstal / install as a plugin?" if yes, all separate files makes sense, otherwise it can probably stay. But I'm not opposed to the removal in general.

Reply:
I have put the License under 'Core'. For some reason, the general ACL file is already in that location, while the general Menu file is under 'Base'.

Since the Base/Menu/Menu.xml file only appears to list legacy pages, I think putting it into its own Menu.xml is the best way to go.

Reply:
why not move base menu file to core?

Reply:
It seemed too important for me to touch, but if you'd like me to, I'll do it.

Reply:
yes please, "Base" prefix predates some later works that have a more aligned structure like ACL.

Reply:
Done.